test/functional/test_framework/test_node.py

```diff
@@ -237,6 +234,9 @@ def wait_for_rpc_connection(self):
                 # This might happen when the RPC server is in warmup, but shut down before the call to getblockcount
                 # succeeds. Try again to properly raise the FailedToStartError
                 pass
+            except OSError as e:
+                if e.errno != errno.ECONNREFUSED:  # Port not yet open?
+                    raise  # unknown IO error
```

Comment:
Super nit: Change comment to say `unknown OS error`?

Reply:
Thanks, fixed